environments/hca-prod/terraform/provider.tf

```diff
@@ -4,12 +4,18 @@ provider google-beta {
   project = local.prod_project_id
   region  = "us-central1"
 }
-provider google-beta {
+provider google{
   alias   = "prod-core"
   project = "broad-dsp-monster-prod"
   region  = "us-central1"
 }
 
+provider google {
```

Comment:
shouldn't this be replacing the provider on line 1, rather than being added as a new provider?